eng/common/scripts/SemVer.ps1

```diff
@@ -4,102 +4,244 @@ Parses a semver version string into its components and supports operations aroun
 
 See https://azure.github.io/azure-sdk/policies_releases.html#package-versioning
```

Comment:
This document does not yet reflect the `beta` convention for .NET and JS

Reply:
That is my PR https://github.com/Azure/azure-sdk/pull/1536